auth_oauth_multi_token/models/res_users.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2016 Florent de Labarre
+# Copyright 2017 Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+import uuid
+
+from odoo import api, fields, models, exceptions
+
+from odoo.addons import base
+base.models.res_users.USER_PRIVATE_FIELDS.\
+    append('oauth_master_uuid')
+
+
+class ResUsers(models.Model):
+    _inherit = 'res.users'
+
+    def _generate_oauth_master_uuid(self):
+        return uuid.uuid4().hex
+
+    oauth_access_token_ids = fields.One2many(
+        comodel_name='auth.oauth.multi.token',
+        inverse_name='user_id',
+        string='OAuth tokens',
+        copy=False,
+        groups='base.group_system'
```

Comment:
what's the specific reason for adding the group now?

Reply:
a standard user have no access to `auth.oauth.multi.token`